infrastructure/terraform/modules/aoai/aoai.tf

```diff
@@ -25,7 +25,7 @@ resource "azurerm_cognitive_account" "cognitive_service" {
     trimsuffix(replace(var.key_vault_uri, "https://", ""), "/")
   ]
   kind               = var.cognitive_service_kind
-  local_auth_enabled = false
+  local_auth_enabled = true
```

Comment:
Are we now using key auth? Why is this required? Please disable if not required.